tests/test_client_response.py

```diff
@@ -1142,7 +1141,7 @@ async def test_response_read_triggers_callback(
     loop: asyncio.AbstractEventLoop, session: ClientSession
 ) -> None:
     trace = mock.Mock()
-    trace.send_response_chunk_received = make_mocked_coro()
+    trace.send_response_chunk_received = mock.AsyncMock()
```

Comment:
Ideally most of these should be replaced with create_autospec(), if anyone is willing to do a bit more on these and help make the code more robust.